src/Nethermind/Nethermind.Serialization.Json/UInt256Converter.cs

```diff
@@ -29,8 +29,18 @@ private static UInt256 ReadInternal(ref Utf8JsonReader reader, JsonTokenType all
     {
         if (reader.TokenType == JsonTokenType.Number)
         {
-            return reader.GetUInt64();
+            if (reader.TryGetUInt64(out ulong smallValue))
+            {
+                return smallValue;
+            }
+
+            ReadOnlySpan<byte> bytes = reader.HasValueSequence ? reader.ValueSequence.ToArray() : reader.ValueSpan;
+            Span<char> chars = stackalloc char[bytes.Length + 1];
```

Comment:
This is Uint256, or can it be something bigger? If Uint256, should we have a fixed stackalloc here maybe? Can be a bit faster.